src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,266 @@
+  optional = true)
+  private Boolean createWorkspace = false;
+
+  @Override
+  public boolean requiresIntervals() { return true; }
+
+  @Override
+  public boolean requiresReads() { return false; }
+
+  @Override
+  public boolean requiresReference() {
+    return false;
+  }
+
+  // The GenomicsDB importer object
+  private GenomicsDBImporter importer = null;
+
+  // Intervals from command line (singleton for now)
+  private List<ChromosomeInterval> intervals;
+
+  /**
+   * From the command line validate
+   * 1. GenomicsDB workspace exists
+   * 2. GenomicsDB workspace is a valid one
+   *
+   * @return  Error messages for the exceptions
+   */
+  @Override
+  protected String[] customCommandLineValidation() {
+    String[] messages = new String[2];
+    int index = 0;
+
+    if (workspace.isEmpty()) {
+      messages[index] = " Must specify a workspace with -GW or --GenomicsDBWorkspace option";
+      return messages;
+    }
+
+    File workspaceDir = new File(workspace);
+
+    if (!createWorkspace) {
+      if (!Files.isDirectory(workspaceDir.toPath()))
+        messages[index++] = "No such directory or workspace found: " + workspace;
+
+      boolean check = new File(workspace + "/__tiledb_workspace.tdb").exists();
+
+      if (Files.isDirectory(workspaceDir.toPath()) && !check) {
+        messages[index++] = "Not a valid GenomicsDB workspace: " + workspace;
+      }
+    }
+    
+    if (sizePerColumnPartitionPerSample < 1024L) {
+      messages[index++] = "Size per column partition per sample is tool small." +
+        " Consider using larger size than 10KBytes";
+    }
+
+    return index==0 ? null : messages;
+  }
+
+  /**
+   * Before traversal, fix configuration parameters and initialize
+   * GenomicsDB. Hard-coded to handle only VCF files and headers
+   */
+  @Override
+  public void onTraversalStart() {
+
+    File workspaceDir = new File(workspace);
+
+    if (createWorkspace) {
+
+      int ret = GenomicsDBImporter.createTileDBWorkspace(workspaceDir.getAbsolutePath());
+
+      if (ret > 0) {
+        try {
+          throw new IOException("Directory " + workspaceDir + " already exists");
+        } catch (IOException e) {
+          e.printStackTrace();
+          return;
+        }
+      } else if (ret < 0) {
+        try {
+          throw new IOException("Error creating GenomicsDB workspace");
+        } catch (IOException e) {
+          e.printStackTrace();
+          return;
+        }
+      }
+    }
+
+    initializeIntervals();
+
+    List<VCFHeader> headers = new ArrayList<>();
+
+    Map<String, FeatureReader<VariantContext>> sampleToVCFMap = new HashMap<>();
+    for (String variantFile : variantFileNames) {
+      String variantFileAbsPath = new File(variantFile).getAbsolutePath();
+      AbstractFeatureReader<VariantContext, LineIterator> reader =
+        AbstractFeatureReader.getFeatureReader(variantFileAbsPath, new VCFCodec(), true);
```

Comment:
To support GCS inputs, you'll need to modify these lines as follows:

```
      final String variantURI = IOUtils.getPath(variantFile).toAbsolutePath().toUri().toString();
      final Function<SeekableByteChannel, SeekableByteChannel> cloudWrapper = (cloudPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudPrefetchBuffer, is) : Function.identity());
      final Function<SeekableByteChannel, SeekableByteChannel> cloudIndexWrapper = (cloudIndexPrefetchBuffer > 0 ? is -> SeekableByteChannelPrefetcher.addPrefetcher(cloudIndexPrefetchBuffer, is) : Function.identity());
      AbstractFeatureReader<VariantContext, LineIterator> reader = AbstractFeatureReader.getFeatureReader(variantURI, null, new VCFCodec(), true, cloudWrapper, cloudIndexWrapper);
```

Should also probably rename `variantFileNames` and `variantFile` to `variantPaths` and `variantPath` in this loop.